examples/preview/theme/front-page.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { usePosts, useGeneralSettings } from '@wpengine/headless';
+import { CTA, Header, Footer, Hero, Posts } from '../components';
+import styles from '../scss/theme/front-page.module.scss';
+
+export default function FrontPage(): JSX.Element {
+  const posts = usePosts();
+  const settings = useGeneralSettings();
+
+  return (
+    <>
+      <Header title={settings?.title} description={settings?.description} />
+      <main className="content">
+        <Hero
+          title="Get Started with Headless"
+          buttonText="Developer Docs"
+          buttonURL="https://developers.wpengine.com/"
+          button2Text="Headless on GitHub"
+          button2URL="https://github.com/wpengine/headless-framework"
+          bgImage="/images/headless_hero_background.jpg"
+          id={styles.home_hero}>
+          <p>
+            WP&nbsp;Engine’s Headless WordPress Framework includes this starter
+            theme, the{' '}
+            <a href="https://github.com/wpengine/headless-framework#plugin-features">
+              headless WordPress plugin
+            </a>
+            ,{' '}
+            <a href="https://www.npmjs.com/package/@wpengine/headless">
+              headless package
+            </a>
+            , and <a href="https://developers.wpengine.com/">tutorials</a> to
+            make building headless WordPress sites fast and fun.
+          </p>
+        </Hero>
+        <section className={styles.explore}>
+          <div className="wrap">
+            <h2>Explore the Starter Theme</h2>
```

Comment:
I think we should be calling this an example project, not a theme at this point

Reply:
Agreed. Fixed in Changed in https://github.com/wpengine/headless-framework/pull/109/commits/63df63cb4c37f4e7113b88f695d91feb413bde82.